SConstruct

```diff
@@ -0,0 +1,9 @@
+VULKAN_SDK = "/home/jeremy/VulkanSDK/1.2.135.0/x86_64"
```

Comment:
Same as above, is the path not retrievable automatically from SDK installation?

Reply:
Removed SConstruct